CIScripts/Tests/VRouterAgentTests.ps1

```diff
@@ -1,8 +1,13 @@
 function Test-VRouterAgentIntegration {
-    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session,
+    Param ([Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session1,
+           [Parameter(Mandatory = $true)] [System.Management.Automation.Runspaces.PSSession] $Session2,
            [Parameter(Mandatory = $true)] [TestConfiguration] $TestConfiguration)
 
     . $PSScriptRoot\CommonTestCode.ps1
+    . $PSScriptRoot\..\Job.ps1
+
+    # TODO: Time tracking should be handled consistently for the whole test suite.
```

Comment:
What about this `TODO`? From what I see, `$AgentIntegrationTestsTimeTracker` is used in all tests in this suite.